### ISSUE TITLE
Create build and CI management 

### ISSUE THREAD

**Issue by sascha15432** [quarterbit/WaterWatcher#2]

**Comment by quarterbit** [comment]
good idea! I need to consider to not have the AWS Certificates on github but as a private part of the deployment

**Comment by quarterbit** [comment]
How can I grant you the right you asked for? I still learn github as a user...

**Comment by sascha15432** [comment]
For sure we will not expose AWS Certificates on Github. I'll sketch a idea first in a the feature branch feature/2. 

For the rights I am not sure, I guess It is located as it is in GitHub enterprise. There it is under "collaboration and teams" within the settings tab of the repo or the organistaion.

**Comment by sascha15432** [comment]
![githubusers](https://user-images.githubusercontent.com/17555993/35315043-8db7e642-00ca-11e8-8d84-b2c051fbeeb5.png)

**Comment by sascha15432** [comment]
See https://github.com/orgs/dev-nil-com/ for helper modules, setting up a CI infrastructure

**Comment by quarterbit** [comment]
please check - rights should be set.

**Comment by sascha15432** [comment]
works for me, I'm going to prepare a proposal for CI and Build mangment
